docs/platform/release-notes/tizen-6-5-m2.md

```diff
@@ -0,0 +1,400 @@
+# Tizen 6.5 Public M2
+
+Release date: Oct. 31, 2021
+
+
+## Release Notes
+
+### System (Kernel and System framework)
+
+#### New and changed features
+
+- New API and features
+  - Kernel
+    - A finegrained version of THP (Transparent Hugepage) in Linux kernel for embedded systems has been implemented in RPI4.
+  - HAL
+    - HAL API Layer is implemented for unifying the HAL backend style and splitting out the hardware dependency.
+    - New image structure with hal.img is adapted for HAL, a set of HAL backend set in /hal.

```

Comment:
This sentence is not clear,  hal.img  is that correct usage?

Would it be okay to include the term path here if that is what is meant?
set in path /hal.

Reply:
+ New image structure is adapted for HAL, new partition is added for a set of HAL backend packages

Reply:
Right.. new line be added and remove the old sentence.